src/main/java/com/manardenza/console/ListMenu.java

```diff
@@ -0,0 +1,59 @@
+package com.manardenza.console;
+
+import lombok.Getter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ListMenu {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1118)